src/components/Trade/Components/TradeQuotes/TradeQuote.tsx

```diff
@@ -218,7 +218,11 @@ export const TradeQuoteLoaded: React.FC<TradeQuoteLoadedProps> = ({
           <RawText color='gray.500'>
             <FaGasPump />
           </RawText>
-          <Amount.Fiat value={networkFeeFiat} />
+          {networkFeeFiat && !networkFeeFiat.isZero() ? (
+            <Amount.Fiat value={networkFeeFiat?.toString()} />
+          ) : (
+            translate('trade.unknownGas')
+          )}
```

Comment:
This will break CoWSwap, where 0 is a valid miner fee (there are no ETH Tx fees for CoW, the fees are subtracted from the sell asset).

We might want to use different heuristics here, and check if the sell amount is currently zero in addition to this check.

Prod:

<img width="555" alt="image" src="https://user-images.githubusercontent.com/17035424/236450444-36b92b46-07db-445a-9a53-ef866cefa02b.png">

This diff:

<img width="562" alt="image" src="https://user-images.githubusercontent.com/17035424/236450556-8fbe3eff-b541-4c98-8dcb-d6689f8b99eb.png">